Doc/c-api/init_config.rst

```diff
@@ -583,6 +583,13 @@ PyConfig
 
       Default: ``0``.
 
+   .. c:member:: int warn_default_encoding
+
+      If equals to 1, emit a ``EncodingWarning`` when ``TextIOWrapper``
+      used its default encoding. See :pep:`597` for detail.
```

Comment:
```suggestion
      If non-zero, emit a :exc:`EncodingWarning` warning when :class:`io.TextIOWrapper`
      uses its default encoding. See :pep:`597` for detail.
      
      Default: ``0``.
```

"equals to 1": the C code uses `if (warn_default_encoding)`, so "non-zero" is more correct.